plugin/pkg/admission/security/podsecuritypolicy/BUILD

```diff
@@ -17,13 +17,13 @@ go_library(
         "//pkg/registry/rbac:go_default_library",
         "//pkg/security/podsecuritypolicy:go_default_library",
         "//pkg/security/podsecuritypolicy/util:go_default_library",
-        "//pkg/serviceaccount:go_default_library",
         "//staging/src/k8s.io/api/policy/v1beta1:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/api/equality:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/labels:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/util/validation/field:go_default_library",
         "//staging/src/k8s.io/apiserver/pkg/admission:go_default_library",
         "//staging/src/k8s.io/apiserver/pkg/admission/initializer:go_default_library",
+        "//staging/src/k8s.io/apiserver/pkg/authentication/serviceaccount:go_default_library",
```

Comment:
Do you have extra white space here?

Reply:
I didn't see an extra space. Would you mind to point out? 🥺

Reply:
Githubs PR view makes it look like its indented one space more that the line above. As long as that's just a viewing artifact all good.